### DISCUSSION TITLE
2023/06/02/opentelemetry-laravel-autoinstrumentation

### DISCUSSION THREAD

**Discussion by giscus** [gulasz101/gulasz101.github.io#11] [Q&A]
# 2023/06/02/opentelemetry-laravel-autoinstrumentation

opentelemetry rc1 released, have to say works really stable! Go for it and test :)

https://gulasz101.github.io/2023/06/02/opentelemetry-laravel-autoinstrumentation.html

**Comment by gulasz101** [comment]
As test, I will simply post here link to status of [opentelemetry for php](https://opentelemetry.io/docs/instrumentation/php/#status-and-releases)
